sdk/tables/Azure.Data.Tables/src/TablesTransactionalBatch.cs

```diff
@@ -144,13 +172,23 @@ public virtual async Task<Response<List<Response>>> SubmitBatchAsync(Cancellatio
         /// </summary>
         /// <typeparam name="T"></typeparam>
         /// <returns></returns>
-        public virtual Response<List<Response>> SubmitBatch<T>(CancellationToken cancellationToken = default)
+        public virtual Response<TablesBatchResponse> SubmitBatch<T>(CancellationToken cancellationToken = default)
```

Comment:
Nit - consider using the shared implementation pattern where you pass `bool async` to determine if you're executing synchronously or not.  This code may grow more complex over time.

Reply:
I'll defer until it does get more complex.